pkg/ccl/importccl/import_stmt.go

```diff
@@ -126,6 +126,10 @@ func importJobDescription(
 func importPlanHook(
 	_ context.Context, stmt tree.Statement, p sql.PlanHookState,
 ) (sql.PlanHookRowFn, sqlbase.ResultColumns, []sql.PlanNode, bool, error) {
+	if !p.ExecCfg().Settings.Version.IsActive(cluster.VersionPartitionedBackup) {
+		return nil, nil, nil, false, errors.Errorf("INSERT requires a cluster fully upgraded to version >= 19.2")
```

Comment:
s/`INSERT`/`IMPORT`/

Reply:
This whole check needs to be below the `importStmt, ok :=` below or it will just error out _every sql statement_ during an upgrade.

Reply:
Done